### DISCUSSION TITLE
Patch release (v0.16.1) with Camel NPE fix

### DISCUSSION THREAD

**Discussion by ghost** [open-telemetry/opentelemetry-java-instrumentation#2260] [Q&A]
There is a recently-fixed issue with possible NPE in the Apache Camel instrumentation (AWS SDK propagation scenario). I'd like to patch release v0.16.0 with this fix as there is an OTEL instrumentation user that is affected. Let me know what you think.

**Comment by anuraaga** [comment]
Critical bug fix seems like a great reason to issue a patch

**Reply by mateuszrzeszutek** [reply]
I'll take care of the patch release then

**Reply by iNikem** [reply]
@mateuszrzeszutek @anuraaga should we include #2259 as well?

**Reply by anuraaga** [reply]
@iNikem Because metrics is alpha I was leaning towards not hotfixing. But since we will release anyways, then yes we may as well put it in :)

**Reply by mateuszrzeszutek** [reply]
Hmm, the release workflow has finished a few minutes ago - @anuraaga should I release 0.16.2 with the prometheus fix?

**Reply by iNikem** [reply]
I vote against :)

**Reply by anuraaga** [reply]
Oh if it's done already then no worries. Thanks @mateuszrzeszutek
